src/commands/recipes/index.ts

```diff
@@ -17,6 +17,10 @@ export const createRecipesCommand = (program: BaseCommand) => {
     .argument('[name]', 'name of the recipe')
     .description(`Create and modify files in a project using pre-defined recipes`)
     .option('-n, --name <name>', 'recipe name to use')
+    .option(
```

Comment:
This will add a flag to the entire `recipes` command, not just this recipe. Right now the command itself is fully decoupled from the individual recipes, by design, and is not aware of any specific recipe. The fact that you're adding a parameter that is specific to a recipe, accompanied by help text that says it is relevant only to a specific recipe, breaks that.

If we keep using this pattern, we'll end up with a myriad of flags, each associated with a given recipe. I think that will get messy.

Do we really need this flag? Can we not just do the selection automatically when we detect thr IDE? If we really want an opt-out mechanism, I would recommend an environment variable as a stopgap solution until we build support for flags into the recipes themselves.